### PR TITLE
Update Psalm baseline

### PR DESCRIPTION
**Pull request by kamil-tekiela**

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -5700,23 +5701,79 @@
             ],
             "time": "2023-11-08T05:53:05+00:00"
         },
+        {
+            "name": "rector/rector",
```
Comment (MauricioFauth) [inline composer.lock:5705]: This is a mistake.

Comment (kamil-tekiela) [reply composer.lock:5705]: ohh good catch, I forgot to remove it

Comment (kamil-tekiela) [reply composer.lock]: Why is composer.lock committed anyway? This file should not be in the git tree.

**Review by MauricioFauth — APPROVED** [review]

File: composer.lock (hunk 1 above)
Comment (MauricioFauth) [reply composer.lock]: One of the reasons is that is easier to investigate issues in old commits, as the dependencies for a specific commit will always be the same.

**Pull request merged**